refactor plugin - increase supported android OS

majeed-platogo opened the pull request:
- remove unused if condition to increase android support
- clean up unused imports
- removed unsed functions

Luro91 commented:
Please also fix the warnings from android studio:
```
The value 'null' assigned to 'js_out' is never used
The value 'null' assigned to 'output' is never used
The value 'null' assigned to 'code' is never used
The value 'null' assigned to 'jpeg_data' is never used
```

![Screenshot 2024-05-29 at 09 24 26](https://github.com/platogo/cordova-screenshot/assets/10352634/83f6f291-53d7-4e17-be27-7895204b3a1a)

majeed-platogo committed:
refactor plugin
- remove unused if condition to increase android support
- clean up unused imports
- removed unsed functions

Co-authored-by: Victoria Kalimullina <victoria@platogo.com>

majeed-platogo commented:
> Please also fix the warnings from android studio:
> 
> ```
> The value 'null' assigned to 'js_out' is never used
> The value 'null' assigned to 'output' is never used
> The value 'null' assigned to 'code' is never used
> The value 'null' assigned to 'jpeg_data' is never used
> ```
> 
> ![Screenshot 2024-05-29 at 09 24 26](https://private-user-images.githubusercontent.com/10352634/334697570-83f6f291-53d7-4e17-be27-7895204b3a1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NzIwNTQsIm5iZiI6MTcxNjk3MTc1NCwicGF0aCI6Ii8xMDM1MjYzNC8zMzQ2OTc1NzAtODNmNmYyOTEtNTNkNy00ZTE3LWJlMjctNzg5NTIwNGIzYTFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI5VDA4MzU1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjNzQyNzlkMzk1N2FlMzJjN2U4OWJiNjEzMTRlNzRhNWQyODg4ZTlmMGNjZjdjZDBkMmRmNTg3NTZhM2QyZGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.D_79MINLN0GpyXizHThZAn0P-DUum37j3OJjkUcJLiU)

done @Luro91

Luro91 reviewed (approved):
Good job @majeed-platogo & @victoriaplatogo
